packages/suite/src/views/wallet/send/components/Outputs/components/Address/index.tsx

```diff
@@ -222,10 +277,28 @@ export const Address = ({ output, outputId, outputsCount }: AddressProps) => {
     });
     const addressBottomText = isAddressWithLabel ? addressLabelComponent : null;
 
+    const getBottomText = () => {
+        if (addressError) {
+            return (
+                <InputError message={addressError.message} button={getValidationButtonProps()} />
+            );
+        }
+
+        if (addressInfoContent) {
+            const mess = 'Converted to Checksum'; //TODO: add translation
+
+            const tooltip = { content: 'Info about checksum conversion' };
+
+            return <InputInfo message={mess} tooltip={tooltip} />;
```

Comment:
You can put both values right inside the component no need to put there in separated consts